src/currentprocess.rs

```diff
@@ -232,6 +231,8 @@ impl TestProcess {
 #[cfg(feature = "test")]
 impl From<TestContext> for TestProcess {
     fn from(inner: TestContext) -> Self {
+        use tracing_subscriber::util::SubscriberInitExt;
```

Comment:
So we import `DefaultGuard` at the top with a `feature = "test"` guard, and import this inside the item where we need it. I think we should try to be consistent? Personally I prefer to have the imports at the top, where they're out of the way of the actual code. What do you think?

Reply:
@djc Okay, reverted!